lms/templates/student_account/login.underscore

```diff
@@ -1,27 +1,60 @@
-<form id="login" class="login-form">
-    <div class="status already-authenticated-msg hidden" aria-hidden="true">
-        <% if (context.currentProvider) { %>
-            <p class="message-copy">
-                <%- _.sprintf(gettext("You've successfully logged into %(currentProvider)s, but your %(currentProvider)s account isn't linked with an %(platformName)s account. To link your accounts, go to your %(platformName)s dashboard."), context) %>
-            </p>
-        <% } %>
+<div class="status already-authenticated-msg hidden">
+    <% if (context.currentProvider) { %>
+        <p class="message-copy">
+            <%- _.sprintf( gettext("You've successfully logged into %(currentProvider)s, but your %(currentProvider)s account isn't linked with an %(platformName)s account. To link your accounts, go to your %(platformName)s dashboard."), context ) %>
```

Comment:
> You've successfully logged into
This should probably read "You've successfully signed into," since we're changing "log in" to "sign in."